### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by guillaume-gomez**

### CODE REVIEW

File: src/signup.erl [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		{ok, Login} -> ValidLogin = Login;
+		_ -> ValidLogin = ""
+	end,
+	case yaws_api:postvar(Arg,"password") of
+		{ok, Password} -> ValidPassword = Password;
+		_ -> ValidPassword = ""
+	end,
+	case user_table:create_user(ValidLogin, ValidPassword) of
+		{atomic,ok} -> return:created([{info, <<"User created !">>}]);
+		_ -> return:not_found([{info, <<"User cannot be created !">>}])
```
Comment (BastienDuplessier) [inline src/signup.erl]: not_found if you can't create user ?

File: src/signup.erl [hunk 2]
```diff
@@ -0,0 +1,34 @@
+-module(signup).
```
Comment (BastienDuplessier) [inline src/signup.erl]: You don't create a "signup". You create a user.
So your route should be `POST users` and not `POST signup`

File: src/signup.erl [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+is_user_exist(Arg) ->
+	case auth:get_user(Arg) of
+        {ok, _User} ->
+        	error;
+        not_found -> 
+        	io:format("TAAA : ~p~n", [ok]),
+            ok
+    end.
+
+create_user(Arg) ->
```
Comment (BastienDuplessier) [inline src/signup.erl]: We need to think about a better design. Maybe use thows and handle exceptions higher.

File: src/signup.erl [hunk 4]
```diff
@@ -0,0 +1,34 @@
+-module(signup).
+-author(["Guillaume Gomez"]).
+-export([create/1]).
+
+
+create(Arg) ->
+	io:format("Has password : ~p~n", [auth:get_user(Arg)]),
```
Comment (BastienDuplessier) [inline src/signup.erl]: Don't forget to get rid of `io:format` in your PRs

File: src/signup.erl (hunk 2 above)
Comment (guillaume-gomez) [inline src/signup.erl]: if a name it "users", how to see the difference between `users index` and `create user` route ?

Comment (ZangtherRGSS3) [inline src/signup.erl]: Index is : `GET users`
Create is : `POST users`

Comment (BastienDuplessier) [inline src/signup.erl]: Users' index will be `GET users` and users' creation will be `POST users`

**Pull request merged**